src/patterns/dosdp-patterns/left_teeth.yaml

```diff
@@ -0,0 +1,91 @@
+pattern_name: right__left_teeth
+pattern_iri: http://purl.obolibrary.org/obo/uberon/patterns/left_teeth.yaml
+
+description: "This pattern defines primary and secondary teeth for the left side of the mouth."
+
+contributors:
+  - https://orcid.org/0000-0001-9625-1899 # Bill Duncan
+
+classes:
+  thing: "owl:Thing"
+  Homo sapiens: "NCBITaxon:9606"
+  calcareous tooth: "UBERON:0001091"
+  jaw region: "UBERON:0011595"
+  maxilla: "UBERON:0002397"
+  mandible: "UBERON:0001684"
+
+relations:
+  in lateral side of: "BSPO:0000126"
+  in right side of: "BSPO:0000121"
+  in left side of: "BSPO:0000120"
+
+annotationProperties:
+  hasExactSynonym: "oboInOwl:hasExactSynonym"
+  present_in_taxon: "RO:0002175"
+  contributor: "http://purl.org/dc/terms/contributor"
+  creation_date: "oboInOwl:creation_date"
+  hasDbXref: "oboInOwl:hasDbXref"
+  hasSynonymType: "oboInOwl:hasSynonymType"
+
+vars:
+  parent_class: "calcareous tooth"       # UBERON:0001091
+  jaw_region: "jaw region"               # UBERON:0011595
+  FMA_label: "xsd:string"                # e.g., right upper third secondary molar tooth
+  human_preferred: "UBERON:core#HUMAN_PREFERRED"
+
+data_vars:
+  laterality: "xsd:string"     # e.g., right, left
+
+data_list_vars:                # separate multiple entries by "|"
+  synonym: "xsd:string"        # e.g., ADA tooth 1, FDI tooth 18
+  creator: "xsd:anyURI"        # e.g., https://orcid.org/0000-0001-9625-1899
+  date_created: "xsd:dateTime" # e.g., 2022-11-13T00:00:00Z; note: this shows up as a string in Protege
+  taxon: "xsd:string"          # e.g., obo:NCBITaxon_9606; note: the full IRI is being used as a value
+  FMA_xref: "xsd:string"       # e.g., FMA:55696
+
+name:
+  text: "%s %s"   # e.g, right upper third secondary molar tooth
+  vars:
+    - laterality   # e.g., right
+    - parent_class # e.g., right upper third secondary molar tooth
+
+def:
+  text: "A %s that is in the right side of the %s."
+  # e.g., A upper third secondary molar tooth that is in the right side of the upper jaw region.
+  vars:
+    - parent_class
+    - jaw_region
+
+# exact_synonym:
+#   value: FMA_label
+#   xrefs: FMA_xref
+# 
+# xref: 
+#   value: FMA_xref
+
+annotations:
+  - annotationProperty: present_in_taxon
+    value: taxon
+  - annotationProperty: hasExactSynonym
+    value: synonym
+  - annotationProperty: contributor
+    value: creator
```

Comment:
change `value` to `var` so it will be generated as an entity in OWL

Reply:
done